ztag/schema.py

```diff
@@ -658,10 +659,13 @@ def ztag_zgrab2_transformed(service, results):
 
 ztag_lookup_axfr = SubRecord({
     "servers":ListOf(SubRecord({
+        "server":String(),
```

Comment:
This should probably be either an FQDN, IPAddress, or a CensysString. What goes into this field?

Reply:
So, I couldn't tell from looking at the code or the error output what is supposed to go in that field. All I could tell was that the records being processed had this field despite the schema missing it. I guessed string based on the occurrence of a `String()`-type server field in another object. I admit this may have been possible to determine and I just missed something.

Reply:
We should look at some of the records in elastic search or run the code to see what comes out.

Reply:
I looked around in some results, and it seems that this value should be an IP address. This is reflected in the newest commit.